src/snyk/common/services/learnService.ts

```diff
@@ -1,146 +1,60 @@
-import axios from 'axios';
-import { isCodeIssue, isCodeIssueOld, isOssIssue, OpenCommandIssueType } from '../../common/commands/types';
-import { SNYK_LEARN_API_CACHE_DURATION_IN_MS } from '../../common/constants/general';
-import type { completeFileSuggestionType } from '../../snykCode/interfaces';
+import vscode from 'vscode';
```

Comment:
You won't be able to unit test this file with this import, but would be good to have tests in place.

Reply:
There's `IVSCodeCommands` inversion for command execution.